README.markdown

```diff
@@ -59,6 +59,21 @@ these terms, please let one of our [core team members](https://jekyllrb.com/team
 
 ## Credits
 
+### Sponsors
+
+Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [Become a sponsor!](https://opencollective.com/jekyll#sponsor)
```

Comment:
We could develop this section a bit, explaining why one should sponsor Jekyll and how this will help the project.

e.g. https://github.com/webpack/webpack#sponsoring